modules/liveIntentIdSystem.js

```diff
@@ -113,7 +112,7 @@ function initializeLiveConnect(configParams) {
 
   // The second param is the storage object, LS & Cookie manipulation uses PBJS
   // The third param is the ajax and pixel object, the ajax and pixel use PBJS
-  liveConnect = liveIntentIdSubmodule.getInitializer()(liveConnectConfig, storage, calls);
+  liveConnect = LiveConnect(liveConnectConfig, storage, calls);
```

Comment:
After thinking more about it, I think we should leave Prebid's default behavior as is and it should use the global bus. When required, later on we can allow to configure the globalVariableName as well as whether the bus should be exposed globally or not.